packages/cli/helpers/copy-prisma-client.ts

```diff
@@ -9,10 +10,11 @@ const clientPath = path.dirname(require.resolve('@prisma/client'))
 const clientPkg = require('@prisma/client/package.json')
 
 // we compute the paths of the files that would get npm published
-const clientFiles = (clientPkg.files ?? []).map((file: string) => path.join(clientPath, file))
+const clientFiles = fg.sync(clientPkg.files, { cwd: clientPath, dot: true, onlyFiles: false })
```

Comment:
```suggestion
// this uses a glob library to understand patterns in files.
// Using tooling from npm would be even better, but it is in magnitudes slower compared to this approach.
// Ideally, it would be great if we could remove this somehow.
const clientFiles = fg.sync(clientPkg.files, { cwd: clientPath, dot: true, onlyFiles: false })
```